plugins/flytekit-sqlalchemy/flytekitplugins/sqlalchemy/task.py

```diff
@@ -54,13 +57,12 @@ def __init__(
     ):
         output_schema = output_schema_type if output_schema_type else FlyteSchema
         outputs = kwtypes(results=output_schema)
-        self._uri = task_config.uri
-        self._connect_args = task_config.connect_args or {}
-        self._secret_connect_args = task_config.secret_connect_args
 
         super().__init__(
             name=name,
             task_config=task_config,
+            container_image="ghcr.io/flyteorg/flytekit:sqlalchemy-latest",
```

Comment:
We probably don't want to use `latest` here right?  Let's say someone has version v0.21 of flytekit, and a task registered with it, and then we update code in the sqlalchemy plugin, and then we release it as flytekit v0.22.  Upon publishing that version, GitHub workflows will build a new sqlalchemy image, and all of a sudden the person who had the existing task would pick up the new image, and behavior may change.

I think it's cleaner to hard code the version.  We can try hard coding the image to the tag that we plan on releasing the image under.  Also kind of weird, but I think it's fine for now.

Reply:
Modified the version name to v0.1

Reply:
@kumare3 / @wild-endeavor

Reply:
Let's not use 0.1.0.  So currently, the thinking is that since plugins are in the same repo as core flytekit code, all the plugins have the same version as core flytekit.  (one GH repo means one version, which I think is quite reasonable.)  So sqlalchemy got published as v0.20.0 recently (see https://pypi.org/search/?q=flytekitplugins).  I think the image version should be the same as the pypi version to avoid confusion.

So basically we should pick the version in the image name to be what we want the release to be.  So if we merge this pr and release into v0.20.1 then that's what the version of the image should be.  I feel like this is a pretty big change to be a patch version, but that's okay.  No one's really using this task anyways, it's just for demo purposes.

Reply:
Modified the version to v0.20.0